packages/ui/src/services/document.service.ts

```diff
@@ -47,7 +48,14 @@ export class DocumentService {
     if (!definition.definitionFiles || Object.keys(definition.definitionFiles).length === 0) return null;
     const content = Object.values(definition.definitionFiles)[0];
     const documentId = definition.documentType === DocumentType.PARAM ? definition.name! : BODY_DOCUMENT_ID;
-    return XmlSchemaDocumentService.createXmlSchemaDocument(definition.documentType, documentId, content);
+    switch (definition.definitionType) {
+      case DocumentDefinitionType.XML_SCHEMA:
+        return XmlSchemaDocumentService.createXmlSchemaDocument(definition.documentType, documentId, content);
+      case DocumentDefinitionType.JSON_SCHEMA:
+        return JsonSchemaDocumentService.createJsonSchemaDocument(definition.documentType, documentId, content);
+      default:
+        return null;
+    }
```

Comment:
Is it possible to add a test for this method to check the resulting object?

Reply:
added :+1:  https://github.com/KaotoIO/kaoto/pull/2298/files#diff-99aa98882b85d66c37412da539dfd9fb8f53826c88c6a258028c87829f1b95a5R12-R67